config/cmake/scripts/HDF4config.cmake

```diff
@@ -29,7 +32,7 @@ cmake_minimum_required (VERSION 3.12)
 ##############################################################################
 
 set (CTEST_SOURCE_VERSION "4.2.17")
-set (CTEST_SOURCE_VERSEXT "-1, currently under development")
+set (CTEST_SOURCE_VERSEXT "-2")
 
```

Comment:
The version for master should still be "currently under development"?  Should it be changing from -1 to -2?